getting-started.rst

```diff
@@ -53,7 +53,7 @@ Autostart
 .. note::
     Autostart is set up automatically by the Windows installer and for Arch Linux by the AUR package (if your desktop environment supports `XDG Autostart <https://wiki.archlinux.org/index.php/XDG_Autostart>`_).
 
-You might want to make ``aw-qt`` start automatically on login using the
```

Comment:
`.profile` isn't the right way to do it and `.xinitrc` is Linux-specific (and also bad practice for most common desktop environments which have "Startup Applications" or similar in settings). 

The sentence should just end after `on login`. If you edit that I'll merge it :)